jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/card/CardKitchenSinkTests.java

```diff
@@ -0,0 +1,116 @@
+package io.github.epam.bootstrap.tests.composite.section.card;
+
+import com.epam.jdi.light.driver.WebDriverFactory;
+import io.github.epam.TestsInit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+
+import static com.epam.jdi.light.elements.composite.WebPage.getUrl;
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.cardKitchenSink;
+import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class CardKitchenSinkTests extends TestsInit {
+    private String imgSrc = "https://jdi-testing.github.io/jdi-light/images/spider-man.jpg";
+    private String imgAlt = "Spider Man";
+    private String titleText = "CARD TITLE";
+    private String cardText = "Some quick example text to build on the card title and make up the bulk of the card's content.";
+    private String item0Text = "Cras justo odio";
+    private String item1Text = "Dapibus ac facilisis in";
+    private String item2Text = "Vestibulum at eros";
+    private String link0Text = "JDI Light Github";
+    private String link1Text = "JDI Website";
+    private String link0Url = "https://github.com/jdi-testing/jdi-light";
+    private String link1Url = "https://jdi-testing.github.io/jdi-light/index.html";
+    private WebElement link0;
+    private WebElement link1;
+    private ArrayList<String> tabs;
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test
+    public void itemsTest() {
+        assertEquals(cardKitchenSink.list.size(), 3);
+        cardKitchenSink.list.get(0).is().text(is(item0Text));
```

Comment:
we should use get(1) instead of get(0) in tests, for instance like here: https://github.com/jdi-testing/jdi-light/pull/654/files